packages/atomic/src/components/result-lists/result-list-common.tsx

```diff
@@ -152,6 +162,16 @@ export class ResultListCommon {
     );
   }
 
+  public focusOnNextNewResult(
+    resultListState: FoldedResultListState | ResultListState
+  ) {
+    if (!this.nextNewResultTarget) {
+      throw "Cannot focus on next new result if nextNewResultTarget isn't defined";
```

Comment:
Is this caught anywhere ? Wouldn't this bubble up to our users even though it's essentially an internal error that they cannot do much about ?

Reply:
It isn't caught anywhere, it's meant as an assert. If an error is thrown here, it means there's an error in our code.